g4f/Provider/bing/create_images.py

```diff
@@ -13,10 +13,10 @@
 from typing import Generator, List, Dict
 
 from ..create_images import CreateImagesProvider
-from ..helper import get_cookies, get_event_loop
+from ..helper import get_cookies
```

Comment:
Consider importing the `get_cookies` function from the `..helper` module directly, instead of importing the entire `..helper` module.